### ISSUE TITLE
Please update the wsl-image to include Podman 5.1

### ISSUE THREAD

**Issue by odockal** [containers/podman-machine-wsl-os#2]
Latest Podman available in the wsl image is 5.0.3, while Podman 5.1 is already available.

**Comment by l0rd** [comment]
TL;DR Cirrus CI build for this repository used to be triggered every hour but isn't anymore. Last build was triggered 3 weeks ago. Someone with enough privileges should re-enable the cron build for this repository.

[Cirrus CI build](https://cirrus-ci.com/build/5612207445114880) was triggered 3 weeks ago by the [latest commit pushed to main branch](https://github.com/containers/podman-machine-wsl-os/commit/a49539b885787738f9798f541c12697f9e4e793f). At that time the latest podman for Fedora 40 was [podman 5.0.3](https://bodhi.fedoraproject.org/updates/?packages=podman). This is also visible in the [Cirrus CI build log](https://api.cirrus-ci.com/v1/task/5215017795059712/logs/update.log).

Podman 5.1.0 was released 2 weeks ago and 5.1.1 last week. That explains why the version included in the image is the 5.0.3.

It also looks like Cirrus CI build task was run [every hour via cron build until the 04/19](https://cirrus-ci.com/github/containers/podman-machine-wsl-os). From the [documentation](https://cirrus-ci.org/guide/writing-tasks/#cron-builds): "Cron builds can be configured on a repository's settings page (not in .cirrus.yml)."

**Comment by l0rd** [comment]
The cron build is now running every 3 hours and I have verified that after a `machine init` the podman server version is 5.1:

```
Client:       Podman Engine
Version:      5.2.0-dev
API Version:  5.2.0-dev
Go Version:   go1.22.3
Git Commit:   dcec81e694c855a1eea0106594928248d74c1a78
Built:        Thu Jun  6 13:47:47 2024
OS/Arch:      windows/amd64

Server:       Podman Engine
Version:      5.1.0
API Version:  5.1.0
Go Version:   go1.22.3
Built:        Wed May 29 02:00:00 2024
OS/Arch:      linux/amd64
```

Thanks @n1hility for re-enabling the cron build!

Closing as fixed.

**Comment by odockal** [comment]
@l0rd thanks! I guess that the reason behind was this https://github.com/containers/podman/issues/22439#issuecomment-2066648326
